src/services/hydra.js

```diff
@@ -0,0 +1,107 @@
+var fetch = require('node-fetch')
```

Comment:
Let's use the ORY Hydra SDK instead: https://www.npmjs.com/package/@oryd/hydra-client